File: deck/seed/code/native/kotlin/runtime/base64.kt
import java.util.Base64

object base64 {
    fun encode(input: String): String = Base64.getEncoder().encodeToString(input.toByteArray(Charsets.UTF_8))
    fun decode(input: String): String = String(Base64.getDecoder().decode(input), Charsets.UTF_8)
}

File: deck/seed/code/native/swift/runtime/base64.swift
import Foundation

enum base64 {
    static func encode(_ input: String) -> String { return Data(input.utf8).base64EncodedString() }
    static func decode(_ input: String) -> String {
        guard let d = Data(base64Encoded: input), let s = String(data: d, encoding: .utf8) else { return "" }
        return s
    }
}
